doc/nrf/ug_nrf91_features.rst

```diff
@@ -160,70 +160,90 @@ The :ref:`nrfxlib:nrf_modem` is released as an OS-independent binary library in
+      In its default configuration, the DFU target library is set to support all the types of FOTA upgrades except full modem firmware upgrades, but you can freely enable or disable the support for specific targets.
 
-To set the LTE band lock, enable the :ref:`lte_lc_readme` library in your project configuration file :file:`prj.conf`, by setting the Kconfig option :kconfig:`CONFIG_LTE_LINK_CONTROL`  to ``y``.
+      In addition, the following requirements apply:
 
-Then, enable the LTE band lock feature and the band lock mask in the configuration file of your project, as follows::
+      * |fota_upgrades_req_mcuboot|
+      * If you want to upgrade the upgradable bootloader, the :ref:`bootloader` must be used (:kconfig:`CONFIG_SECURE_BOOT`).
+      * If you want to upgrade the modem firmware through modem delta updates, neither MCUboot nor the immutable bootloader are required, because the modem firmware upgrade is handled by the modem itself.
+      * If you want to perform a full modem firmware upgrade, an |external_flash_size| is required.
 
-   CONFIG_LTE_LOCK_BANDS=y
-   CONFIG_LTE_LOCK_BAND_MASK="10000001000000001100"
+#. Create a binary file that contains the new image.
 
-The band lock mask allows you to set the bands on which you want the modem to operate.
-Each bit in the :kconfig:`CONFIG_LTE_LOCK_BAND_MASK` option represents one band.
-The maximum length of the string is 88 characters (bit string, 88 bits).
+      .. note::
+         This step does not apply for upgrades of the modem firmware.
+         You can download delta patches and full binaries of the modem firmware from the `nRF9160 product website (compatible downloads)`_.
 
-The band lock is a non-volatile setting that must be set before activating the modem.
-It disappears when the modem is reset.
-To prevent this, you can set the modem in *power off* mode, by either:
+      |fota_upgrades_building|
+      The :file:`app_update.bin` file is the file that should be uploaded to the server.
 
-* Sending the AT command ``AT+CFUN=0`` directly.
-* Calling the :c:func:`lte_lc_power_off` function while the *LTE Link Control Library* is enabled.
+      To create binary files for a bootloader upgrade, make sure that :kconfig:`CONFIG_SECURE_BOOT` and :kconfig:`CONFIG_BUILD_S1_VARIANT` are enabled and build MCUboot as usual.
+      The build will create a binary file for each variant of the upgradable bootloader, one for each bootloader slot.
+      See :ref:`upgradable_bootloader` for more information.
 
-Both these options save the configurations and historical data in the Non-Volatile Storage before powering off the modem.
+#. Make the binary file (or files) available for download.
+     Upload the serialized :file:`.cbor` binary file or files to a web server that is compatible with the :ref:`lib_download_client` library.
+     One way of doing this is to upload the files to an Amazon Web Services Simple Storage Service (AWS S3) bucket.
```

Comment:
Another way is to use \`nRF Cloud\`\_.  The files can be uploaded to your nRF Cloud account (\`Creating an nRF Cloud account\`\_) as a bundle on the Device Management / Firmware Updates page.

See  https://docs.nrfcloud.com/Guides/GettingStarted/FOTA [should be added to links.txt]